apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -148,6 +127,54 @@ function LandingPage({
     />
   );
 
+  // Renders at most one banner for a user:
+  // - if the user hasn't used any PL resources yet, show the Getting Started banner
+  // - if the user has a pending workshop survery (mutually exclusive from the above), show a banner to fill out the survey
+  // - else, render either nothing or an announcement banner
+  const RenderBanner = () => {
+    const showGettingStartedBanner =
+      !currentYearApplicationId &&
+      !hasEnrolledInWorkshop &&
+      plCoursesStarted?.length === 0;
+
+    if (showGettingStartedBanner) {
+      return (
+        <TwoColumnActionBlock
+          imageUrl={pegasus(
+            '/images/fill-540x300/professional-learning/pl-superhero-girl-crop.png'
+          )}
+          heading={i18n.plLandingGettingStartedHeading()}
+          subHeading={i18n.plLandingGettingStartedSubHeading()}
+          description={i18n.plLandingGettingStartedDescription()}
+          buttons={[
+            {
+              url: pegasus('/educate/professional-learning'),
+              text: i18n.plLandingGettingStartedButton(),
+            },
+          ]}
+        />
+      );
+    } else if (lastWorkshopSurveyUrl) {
+      return RenderLastWorkshopSurveyBanner();
+    } else if (!!DCDO.get('curriculum-launch-2024', false)) {
+      return (
```

Comment:
Not blocking suggestion: can you add a TODO clean up comment here so we know to remove this eventually after the launch?

Reply:
Done!